pkg/tfbridge/schema_test.go

```diff
@@ -3082,7 +3098,7 @@ func Test_makeTerraformInputsNoDefaults(t *testing.T) {
 				},
 			}),
 			//nolint:lll
-			expect: autogold.Expect(map[string]interface{}{"property_a": "a", "property_b": true, "property_c": map[string]interface{}{"nested_property_a": true}}),
+			expect: autogold.Expect(map[string]interface{}{"property_a": "a", "property_b": true, "property_c": map[string]interface{}{"nestedPropertyA": true}}),
```

Comment:
This is an observable change, and the tests is showing it. map_property_value indicates a Map with unknown element types, which is supported in TF, and the change is that the keys are now passed as-is instead of inflected to Pulumi name notation, which makes perfect sense.